docs/en/enterprise-edition/content-collections/application-security/get-started/connect-code-and-build-providers/third-party-ingestion/sarif-ingestion.adoc

```diff
@@ -0,0 +1,162 @@
+[.task]
+== Ingest SARIF Data
+
+//You can import and consolidate findings from Static Application Security Testing (SAST) vendors that support SARIF output. This enables you to view all your Application Security (AppSec) vulnerabilities on a single platform, whether they are from third-party sources or native tools.
+
+
+Upload SAST data from third-party tools (for vendors that support SARIF output) to the Prisma Cloud platform using SARIF (Static Analysis Results Interchange Format) to analyze and remediate SAST issues directly within the platform. This feature is particularly useful for vendors that are not directly supported by Prisma Cloud. When SARIF data findings are uploaded, they are parsed to generate CWE weaknesses. 
+
+NOTE: If you are using xref:veracode-ingestion.adoc[Veracode], consider using the native ingestion feature instead. The native integration includes automatically pulling the data periodically and provides additional fidelity.
+
+
+Prisma Cloud supports SARIF versions 2.0 and 2.1 and accepts `.zip`, `.json`, and `.sarif` files.
+
+You can upload SARIF files through the Prisma Cloud platform or via the API.
+
+
+. Before you begin:
++
+* Permissions:
++
+** Grant *System Administrator* or *AppSec* privileges to users who need to view and upload SARIF results. Limit access to read-only for selected repositories by assigning the *Developer* role permissions to users who only need to view and analyze data in specific repositories without the ability to make changes
++
+** *GRBAC (Generic Role-Based Access Control)*: Viewing will require permissions for Projects > View and access to the selected repositories. Ingesting SARIF files will require permissions for Providers > Create or Update. Deleting old SARIF files will require permissions for Providers > Delete 
+* Onboard the repository into the system before SARIF findings for that repository can be uploaded
+* Only upload findings relevant to the repository 
+* Ensure that uploaded findings follow the standard SARIF specifications
+* Add the Prisma Cloud IP addresses and hostname for Application Security to an xref:../../../../get-started/console-prerequisites.adoc[allow list] to enable access to the Prisma Cloud console
+
+=== Ingest SARIF via the console
+[.procedure]
+
+. Export the required SARIF file from your third-party vendor to your host machine.
+
+. In Application Security, select *Repositories* > click on a repository > *Add SARIF Findings*.
+
+. Browse for and select the SARIF file from the displayed dialog box.
++
+The file is uploaded and the ingestion process begins (up to ten minutes depending on the size of the ingested data). Popups indicate if ingestion is successful. 
```

Comment:
```suggestion
The file is uploaded and the ingestion process begins (up to ten minutes depending on the size of the ingested data). Popups will notify you that the ingestion upload was successful and that the ingestion processing has beghin.
```

We need to let users know the popup is just saying the upload was successful, not that the ingestion is complete.